### PR TITLE
chore(docs): remove leading dollar sign

### PR DESCRIPTION
**Pull request by nhedger**
See https://github.com/reactphp/http/pull/458

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -548,13 +548,13 @@ To run the test suite, you first need to clone this repo and then install all
...
 ```bash
-$ composer install
+composer install
 ```
 
 To run the test suite, go to the project root and run:
 
 ```bash
-$ php vendor/bin/phpunit
+php vendor/bin/phpunit
```
Comment (clue) [inline README.md]: Looks like a good opportunity to also get rid of the `php` binary here for consistency reasons (not needed and also not used in most other components) :+1:

**Review by clue — COMMENTED** [review]
@nhedger Thank you!

Can you apply/squash this minor suggestion? Once this is in, we should also apply the same changes to the `3.x` and `2.x` branches :+1:

File: README.md (hunk 1 above)
Comment (nhedger) [reply README.md]: Sounds good, will do.

**Review by clue — APPROVED** [review]
@nhedger Thanks for the quick update, changes LGTM! :+1:

**Review by WyriHaximus — APPROVED** [review]

**Comment by clue** [comment]
@nhedger Glad this has been merged! Can you also apply the same changes to the `3.x` and `2.x` branches? :+1:

**Comment by nhedger** [comment]
@clue, see #52 and #53

**Pull request merged**